source/slang/slang-emit-cuda.h

```diff
@@ -76,6 +76,9 @@ class CUDASourceEmitter : public CPPSourceEmitter
     
     SlangResult _calcCUDATextureTypeName(IRTextureTypeBase* texType, StringBuilder& outName);
 
+    void _emitInitArray(IRType* elementType, IRUse* operands, Index operandCount);
+    void _emitInitArrayValue(IRType* elementType, IRInst* value);
```

Comment:
These could arguably be hoisted up to `CLikeSourceEmitter` since initializer lists are a common feature across many of these languages.